cmd/admin-profiling-start.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Minio Client (C) 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"strings"
+
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var adminProfilingStartFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "type",
+		Usage: "Profiler type, possible values are: `cpu`, `mem`, `block`, `mutex` and `trace`",
+		Value: "cpu",
```

Comment:
> should this be `value` instead of `cpu`?

This means "cpu" is the value by default if the user doesn't specify a profiler type using --type flag in `mc admin profiling start` command

Reply:
I think `block` should be default @vadmeste or `mem` - we don't use much of `cpu` ever.

Reply:
> I think `block` should be default @vadmeste or `mem` - we don't use much of `cpu` ever.

Done